modules/migration-compatibility-guidelines.adoc

```diff
@@ -8,18 +8,45 @@
 [id="migration-compatibility-guidelines_{context}"]
 = Compatibility guidelines
 
+// Do we have a "What's New" section in the Release Notes? I'm wondering what, given the complicated support model we discuss here, is the motivation to upgrade from MTC 1.5/1.6 to MTC 1.7? Or is the usage model only: Install once, migrate, and that's it, you don't need MTC after that, so no need to upgrade...?
+
 You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
 
-You cannot install {mtc-short} {mtc-version} on {product-title} 4.5, or earlier versions, because the custom resource definition API versions are incompatible.
+.Restrictions
+
+{product-title} 4.5 and earlier are referred to as _legacy platforms_. {product-title} 4.6 and later are referred to as _modern platforms_.
+
+* Legacy platforms are no longer supported.
```

Comment:
I don't understand this statement. Is this in reference to supported by openshift? Or supported by MTC?

It's not accurate to say that MTC does not support legacy platforms.